### PR TITLE
test on github actions workflow

### PR DESCRIPTION
**Pull request by yseto**
We will migrate GitHub Actions from CircleCI.
therefore I write an execution a tests at workflow.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: ci
+
+on:
+  pull_request: {}
+  push:
+    branches:
+    - master
+#   tags:
+#   - v*
```
Comment (susisu) [inline .github/workflows/ci.yml:10]: Will this be enabled later?

**Review by susisu — APPROVED** [review]
Anyway LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yseto) [reply .github/workflows/ci.yml:10]: yes. :bow:

**Comment by yseto** [comment]
Thanks.

**Pull request merged**